bin/gem2rpm

```diff
@@ -45,6 +60,11 @@ end
 srpmdir = nil
 specfile = nil
 if options[:srpm]
+  unless File.exist?(gemfile)
```

Comment:
You are right, the `$ gem2rpm --srpm gem2rpm` fails with exception. However:

1. This is probably not the right place for this check, since the gemfile is expected to be present also on other places, so the check should be more generic.
2. This is out of scope for this PR IMO. Would you mind to remove the check from this PR entirely and open separate ticket/PR to fix this issue(s)? Please note that there are also other unhandled exceptions, such as requesting to fetch non-existing gem, etc.